src/playersElo.js

```diff
@@ -1,15 +1,21 @@
 import { getPersonalScoreboard } from "./scoreboard";
 
-let url = "https://haxrecordings.s3.amazonaws.com/scoreboard.json";
+const playersElo = async function () {
+  const url = process.env.BASE_API_URL + "/players";
+  let elos = {};
+  await fetch(url)
+    .then(data => data.json())
+    .then(players => {
+      players.forEach(player => {
+        elos[player.name] = player.elo;
+      });
+    });
```

Comment:
If you use `.then` you don't need `await` I think. Is one or the other. With `await` this will look like:

```
const request = await fetch(url)
const players = request.json();

players.forEach(player => {
  elos[player.name] = player.elo;
});

return elos;
```

Reply:
pero `.json()` retorna una promise. por eso le puse otro then al final, y el await la verdad no estoy seguro, pero lo probé así y anduvo todo bien